README.md

```diff
@@ -951,6 +951,45 @@ when you need to retrieve a single record by some attributes.
   end
   ```
 
+* <a name="reversible-migration"></a>
+  Migration commands in the `change` method should write only reversible.
```

Comment:
Don't use non-reversible migration commands in the `change` method. (sounds better this way)